### PR TITLE
New admin user group

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -82,6 +82,7 @@ Users:
 - Hide password in change password view [#3417].
 - Added a change presence view [#3496].
 - New feature to send invitation emails with OpenSlides login [#3503].
+- New admin user group.
```
Comment (normanjaeckel) [inline CHANGELOG]: PR number is missing.

**Comment by emanuelschuetze** [comment]
>  Please also check migration path (that everything works as expected).

I tried the migration from 2.1.1 database to this branch. The admin group was not created because there is no migration file. But it works as expected for new databased. For me this is ok.
@normanjaeckel @donovaly @FinnStutzenstein Ok for you, too? Can I merge it?

**Comment by donovaly** [comment]
OK for me.

**Comment by emanuelschuetze** [comment]
Ok, I'll merge it now.

**Comment by normanjaeckel** [comment]
Ok.

**Pull request merged**